sdk/core/azure-core-serializer-json-gson/src/test/java/com/azure/core/serializer/json/gson/implementation/GsonJsonWriterContractTests.java

```diff
@@ -24,6 +25,18 @@ public void beforeEach() throws IOException {
         this.writer = AzureJsonUtils.createWriter(outputStream, null);
     }
 
+    @AfterEach
+    public void afterEach() throws IOException {
+        if (writer != null) {
+            try {
+                writer.close();
+            } catch (IllegalStateException ignored) {
+                // Closing the JsonWriter may through an IllegalStateException if the current writing state isn't
```

Comment:
```suggestion
                // Closing the JsonWriter may throw an IllegalStateException if the current writing state isn't
```

Reply:
Since this was a copy and paste this exists elsewhere, I'll fix those too